engine/src/main/java/com/datatorrent/stram/StreamingContainerManager.java

```diff
@@ -2386,6 +2399,9 @@ public void deploy(Set<PTContainer> releaseContainers, Collection<PTOperator> un
   @Override
   public void recordEventAsync(StramEvent ev)
   {
+    if (apexPluginManager != null) {
```

Comment:
Currently no. Plugin framework is itself a message bus. we could provide a hook to forward messages received on the bus to our plugin infrastructure. But currently only events are sent on the bus are events and they are already handled by plugin framework. In-fact the eventrecorder could be made as a plugin which needs this eventBus.